src/main/java/net/pod/peaengine/window/Window.java

```diff
@@ -51,19 +52,43 @@ private void init() {
         keyCallback = new GLFWKeyCallback() {
             @Override
             public void invoke(long window, int key, int scancode, int action, int mods) {
-                System.out.println("Key press!");
-                //TODO key press events and shit
+                System.out.println("Key press! " + key);
+                KeyManager.setKey(key, action == GLFW.GLFW_PRESS);
             }
         };
         GLFW.glfwSetKeyCallback(window, keyCallback);
-        mouseButtonCallback = new GLFWMouseButtonCallback() {
+        mouseButtonCallback =new GLFWMouseButtonCallback() {
             @Override
-            public void invoke(long l, int i, int i1, int i2) {
-                System.out.println("Mouse event!");
-                // TODO mouse press events and shit
+            public void invoke(long window, int button, int action, int mods) {
+                System.out.println("Mouse event! " + button);
+                MouseManager.setBtn(button, action == GLFW.GLFW_PRESS);
```

Comment:
needs a fix here that we discussed in discord (about that it sends GLFW_HOLD instead of PRESS after holding the key)